Rebuild-Black-Business/RBB-Website
# Render Atlanta link in About

# Describe your PR
Render Atlanta should be linked in /about

Related to #196 
Fixes #196 

## Pages/Interfaces that will change
/about founder message

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

## Steps to test

1. go to /about
2. verify that Render Atlanta is linked
3. verify that the link works

### Additional notes
We will probably need to refactor how we insert the copy in the future.
